src/main/java/codeu/controller/ServerStartupListener.java

```diff
@@ -36,6 +45,10 @@ public void contextInitialized(ServletContextEvent sce) {
       List<Event> events = PersistentStorageAgent.getInstance().loadEvents();
       EventStore.getInstance().setEvents(events);
 
+      NemoBot nemoBot = new NemoBot();
+      botController.registerBot("@NemoBot", nemoBot);
```

Comment:
Can `registerBot` take care of adding it as a user too?

Reply:
I tried this and it doesn't work. I get the error `incompatible types: codeu.model.data.Bot cannot be converted to codeu.model.data.User`.

Reply:
Never mind, casting `Bot` as `User` worked.

Reply:
However, the tests now say `No API environment is registered for this thread.`

Reply:
That seems like a weird error for such change. Have you googled around?

Reply:
Seems like you've fixed this in the other PR? I can see the bot is registering itself as a user there.